components/tidb_query/src/rpn_expr/impl_op.rs

```diff
@@ -55,6 +59,71 @@ pub fn unary_not_decimal(arg: &Option<Decimal>) -> Result<Option<i64>> {
     Ok(arg.as_ref().map(|v| v.is_zero() as i64))
 }
 
+pub fn map_unary_minus_int_func(value: ScalarFuncSig, children: &[Expr]) -> Result<RpnFnMeta> {
```

Comment:
Prefer to put the map function inside `rpn_expr/mod.rs`

Reply:
Okay